### ISSUE TITLE
can't build on Windows - "'cp' is not recognized as an internal or external command,"

### ISSUE THREAD

**Issue by doehrm** [open-watcom/open-watcom-v2#741]
While trying to build - including the help - I get the following error, no matter which compiler is used:

Watcom:
```
===== htmlhelp D:\_buildWatcom\watcom\open-watcom-v2\docs\htmlhelp-c_readme/c_readme =====
'cp' is not recognized as an internal or external command,
operable program or batch file.
Error(E42): Last command making (c_readme.chm) returned a bad status
Error(E02): Make execution terminated
Error(E42): Last command making (c_readme) returned a bad status
Error(E02): Make execution terminated
<pmake -d build          -h> => non-zero return: 2
Build failed
```

MSVC:
```
===== htmlhelp d:\watcom\open-watcom-v2\docs\htmlhelp-c_readme/c_readme =====
'cp' is not recognized as an internal or external command,
operable program or batch file.
Error(E42): Last command making (c_readme.chm) returned a bad status
Error(E02): Make execution terminated
Error(E42): Last command making (c_readme) returned a bad status
Error(E02): Make execution terminated
<pmake -d build          -h> => non-zero return: 2
Build failed
```

Intel XE2020 has the same error. 

What am I missing?

**Comment by doehrm** [comment]
working now, that was fast, thank you!

**Comment by jmalak** [comment]
Oh sorry. It is fixed by commit 62bb7283
I forgot that installer build make file contains cp command.
In Azure pipelines it is masked by NIX* environment for Windows therefore it is working even if cp normaly doesn't exist on Windows system.

**Comment by jmalak** [comment]
I checked it on fresh system and all is going correctly.
